From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alpha3 <noreply@alpha3.invalid>
Date: Sat, 24 Oct 2026 20:07:39 -0400
Subject: [PATCH 09/17] Android: allow the pass normals buffer of the post
 processor

The post processor switched glDisablei off on Android, which turns the normals
attachment off for every shader. OpenGL ES 3.2 and EXT_draw_buffers_indexed have it.
---
 apps/openmw/mwrender/postprocessor.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/apps/openmw/mwrender/postprocessor.cpp b/apps/openmw/mwrender/postprocessor.cpp
index 6a3ce95073..a196ded347 100644
--- a/apps/openmw/mwrender/postprocessor.cpp
+++ b/apps/openmw/mwrender/postprocessor.cpp
@@ -207,10 +207,6 @@ namespace MWRender
         if (!ext->glDisablei && ext->glDisableIndexedEXT)
             ext->glDisablei = ext->glDisableIndexedEXT;
 
-#ifdef ANDROID
-        ext->glDisablei = nullptr;
-#endif
-
         if (ext->glDisablei)
             mNormalsSupported = true;
         else
